modules/checking-mco-node-status.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-mco-node-status_{context}"]
+= Checking machine config update status
+
+During upgrades you might want to monitor the progress of individual nodes in the case that issues arise and you need to troubleshoot a node.
```

Comment:
do we want to mention that this is behind tech-preview featuregate?

Reply:
yes, there's an include statement a few lines down that pulls in the tech preview snippet we use across docs